workspaces/azure-devops/plugins/scaffolder-backend-module-azure-devops/src/actions/helpers.ts

```diff
@@ -0,0 +1,219 @@
+/*
+ * Copyright 2025 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { LoggerService } from '@backstage/backend-plugin-api';
+import * as azdev from 'azure-devops-node-api';
+import * as GitApi from 'azure-devops-node-api/GitApi';
+import * as GitInterfaces from 'azure-devops-node-api/interfaces/GitInterfaces';
```

Comment:
Can we avoid doing these `import *` imports? It's not a pattern we are using for this plugin so far, would like to keep it this way 👍